slackdump/slackdump.py

```diff
@@ -0,0 +1,62 @@
+import time
+import hashlib
+
+from slackclient import SlackClient
+from pymongo import MongoClient
+
+from .slackhistory import SlackHistory
+
+CONNECTION_SLEEP_INTERVAL = 1
+EMPTY_READ_SLEEP_INTERVAL = .1
+
+
+class SlackDump(object):
+    def __init__(self, auth_token):
+        self.auth_token = auth_token
+        self.slack = SlackClient(self.auth_token)
+        self.client = MongoClient()
+        self.db = self.client.slackdb
+
+    def make_connection(self):
+        try:
+            while not self.slack.rtm_connect():
+                time.sleep(CONNECTION_SLEEP_INTERVAL)
+            return True
+
+        except Exception as ex:
+            return ex
+
+    def get_history(self, connect_time):
+        history = SlackHistory(auth_token=self.auth_token,
+                               connect_time=connect_time)
+        history.start()
+
+    def real_time_message(self):
+        while True:
+            msg = self.slack.rtm_read()
+
```

Comment:
@UmangThapliyal @ManishaBayya Here also we don't need the extra line. Whenever the loop starts no need to specify the extra line. If the loop ends we need to specify the extra line. Please follow the same thing everywhere.